daemon/read.go

```diff
@@ -1,14 +1,13 @@
 package daemon
 
 // import "fmt"
-
+var InstCount int = 2
 func Read() (int, error) {
```

Comment:
what do you think about changing the int to uint ? 

since port is only using +ve side of integers , Uint might be suitable, also according to the required range you can select b/w uint8 or uint16 

no need to change now . just as a suggestion